ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.18.yaml

```diff
@@ -299,6 +299,13 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-azure-ipi-private-sshkey
+- as: azure-ipi-valid-duplicate-dns-f14
```

Comment:
Can we use arm arch to run the testing ?

Reply:
sure, updated.